### ISSUE TITLE
iOS: The iOS touching does not work

### ISSUE THREAD

**Issue by emoonadev** [koush/vysor.io#1122]
Hi,

**The bug report can't be generated**

The mirroring work fine, but the touching does not work at all. I enabled the accessibility `AssistiveTouch`, but Vysor no track the mouse and no respond to the mouse clicks

**Desktop (please complete the following information):**
 - OS: MacOS 12.4

**Comment by emoonadev** [comment]
Ok I understant that the feature broken since Monterey. Fortunately I have at home a NFR dongle, I did all the setup. And now it's work but I have a minor delay. There is a way to improve this delay?

**Comment by koush** [comment]
The delay is unavoidable. Some USB ports are better latency than others.
